src/module/apps/sidebar/encounter-tracker.ts

```diff
@@ -68,6 +68,13 @@ export class EncounterTrackerPF2e<TEncounter extends EncounterPF2e | null> exten
 
         const encounter = this.viewed;
         if (!encounter) return super.activateListeners($html);
+        const trackerParent = tracker.parentElement;
+        const trackerPlaceholder = document.createElement("div");
+
+        if (trackerParent) {
+            trackerParent.insertBefore(trackerPlaceholder, tracker);
+            trackerParent.removeChild(tracker);
+        }
```

Comment:
o here too!

Reply:
In that case we don't even need the trackerParent reference anymore, do we?
Just two replaces, which are still valid and do not throw exceptions even if the element has no parent.
```js
const div = document.createElement('div');
const span = document.createElement('span');
div.replaceWith(span); // all good!
```